ansible/roles/debops.prosody/meta/ansigenome.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
The `meta/ansigenome.yml` files are not needed anymore, since roles don't need their own `README` files.